_docs/response-templating.md

```diff
@@ -30,6 +30,24 @@ WireMockServer wm =
 
 See [the command line docs](../standalone/java-jar/#command-line-options) for the standalone equivalents of these parameters.
 
+Response templating can also be disabled on a per-stub basis by adding the `disableBodyFileTemplating` parameter to the 
```

Comment:
Probably worth making it clear that it's just when it's a body file, not online bodies or any other response element.

Reply:
Ah, thanks for that.  Didn't realise that was the case